Sources/Proxy/Core/PiecewiseFunctionProxy/index.js

```diff
@@ -11,17 +39,86 @@ function vtkPiecewiseFunctionProxy(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkPiecewiseFunctionProxy');
 
+  // Copy to not keep references
+  model.gaussians = model.gaussians.slice();
+  model.points = model.points.slice();
+  model.nodes = model.nodes.slice();
+  model.dataRange = model.dataRange.slice();
+
```

Comment:
This is already done inside the macro.safeArray called by macro.obj (I think)

Reply:
Cool! Checked it out, and it's there. That's good to know.